volume/ceph/ceph.go

```diff
@@ -15,7 +17,11 @@ import (
 	"github.com/opencontainers/runc/libcontainer/utils"
 )
 
-const CephImageSizeMB = 1024 * 1024 // 1TB
+const (
+	CephImageSizeMB   = 1024 * 1024 // 1TB
+	LuksDevMapperPath = "/dev/mapper/"
+	criptoLUKS        = "crypto_LUKS"
```

Comment:
Fixed in https://github.com/medallia/docker/pull/15/commits/e2be6b11bf90ac40363bb3e4c80a28503acbed0c.